setup.py

```diff
@@ -48,8 +48,8 @@ def is_release():
 if not is_release():
     name += '-dev'
     version = f'{parse_version(__version__)}.dev{build_number}'
-    description = 'Dapr SDK for python'
-    long_description = 'This is the development build.'
+    description = 'The developmental release for Dapr Python SDK.'
```

Comment:
@amanbha I changed the description of pkg. once this pr is merged, this description will be shown in pypi pkg.